packages/website/src/bin/default-config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     name: 'Docs',
     description: 'View the documentation for this project',
   },
-  packages: ['./packages/*'],
+  packages: ['./packages/**/*'],
```

Comment:
Can we call out this change in a changeset?

WHAT the change is
WHY the change was made
HOW a consumer should update their code

In this case, the consumer (may) need to update their code.